src/ray/gcs/format/gcs.fbs

```diff
@@ -22,6 +22,7 @@ enum TablePrefix:int {
   TASK_LEASE,
   ACTOR_CHECKPOINT,
   ACTOR_CHECKPOINT_ID,
+  DYNAMIC_RESOURCE,
```

Comment:
maybe name this `node_resource`? 
I feel `node_resource` explains more clearly that this table stores the resources for each node.

Reply:
In that case, should we change the `dynamic_resource_table` to `resource_table`?

Reply:
Yes.